CoreGraphFramework/nbproject/project.xml

```diff
@@ -88,6 +88,7 @@
                 <package>au.gov.asd.tac.constellation.graph.attribute.interaction</package>
                 <package>au.gov.asd.tac.constellation.graph.attribute.io</package>
                 <package>au.gov.asd.tac.constellation.graph.construction</package>
+                <package>au.gov.asd.tac.constellation.graph.hittest</package>
```

Comment:
Does adding the hittest package change the behaviour/code for the jogl renderer?

Reply:
I haven't tested but I can't see how moving files would effect anything.